castai/data_source_gke.go

```diff
@@ -8,13 +8,37 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+const (
+	// GKEPoliciesResourceName is the name of the resource
+	GKEPoliciesResourceName = "policy"
+	// GKEFeaturesResourceName is the name of the policies per feature
+	GKEFeaturesResourceName                        = "features"
+	GKELoadBalancersNetworkEndpointGroupFeature    = "load_balancers_network_endpoint_group"
+	GKELoadBalancersTargetBackendPoolsFeature      = "load_balancers_target_backend_pools"
+	GKELoadBalancersUnmanagedInstanceGroupsFeature = "load_balancers_unmanaged_instance_groups"
```

Comment:
Small thing. No need to use capital letters because it doesn't need to be exported.
```suggestion
const (
	// fieldGKEPolicies is the name of the resource
	fieldGKEPolicies = "policy"
	// fieldGKEFeatures is the name of the policies per feature
	fieldGKEFeatures                        = "features"
	gkeLoadBalancersNetworkEndpointGroupFeature    = "load_balancers_network_endpoint_group"
	gkeLoadBalancersTargetBackendPoolsFeature      = "load_balancers_target_backend_pools"
	gkeLoadBalancersUnmanagedInstanceGroupsFeature = "load_balancers_unmanaged_instance_groups"
```